### PR TITLE
blog post: 18F's engineering craft practices at 10

### PR DESCRIPTION
**Pull request by cantsin**
# Pull request summary

Adds a new blog post: `18F's engineering craft best practices at 10`

### CODE REVIEW

**Comment by cantsin** [comment]
CF https://github.com/18F/blog-drafts/issues/902

**Comment by claireb-gsa** [comment]
@loganmeetsworld the post is just waiting on an image, correct?

**Comment by loganmeetsworld** [comment]
Correct @claireb-gsa! I plan on updating and publishing this on Monday.

**Review by claireb-gsa — APPROVED** [review]
Added the image

File: _posts/2024-06-27-18fs-engineering-craft-basics-at-10.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+---
+title: 18F's engineering craft practices at 10
+date: 2024-06-27
```
Comment (loganmcdonald-noaa) [inline _posts/2024-06-27-18fs-engineering-craft-basics-at-10.md:3]: Let's change this to `2024-07-09` and publish!

**Review by claireb-gsa — APPROVED** [review]
updated publish date

**Pull request merged**